llvm/lib/Target/ARM/ARMInstructionSelector.cpp

```diff
@@ -1103,6 +1103,26 @@ bool ARMInstructionSelector::select(MachineInstr &I) {
     assert((ValSize != 64 || STI.hasVFP2Base()) &&
            "Don't know how to load/store 64-bit value without VFP");
 
+    MachineInstr *Ptr = MRI.getVRegDef(I.getOperand(1).getReg());
```

Comment:
It may make sense to cast MI to GLoad and use helper methods rather than `getOperand(number)`.
There is also GPtrAdd.